runtime/bcutil/cfreader.c

```diff
@@ -2320,13 +2299,7 @@ checkAttributes(J9PortLibrary* portLib, J9CfrClassFile* classfile, J9CfrAttribut
 						goto _errorFound;
 					}
 					/* Capture the error if the outer_class_info_index points to an array class or other referencetype descriptor */
-					if (
```

Comment:
The comment above needs to be updated.

Reply:
> /* Capture the error if the outer_class_info_index points to an array class or other referencetype descriptor */

This comment still mentions "referencetype descriptor", it should be removed.

Reply:
It was changed to `/* Capture the error if the outer_class_info_index points to an array class */`

Reply:
OK. I see that now.